src/components/bottom-bar/bottom-bar.tsx

```diff
@@ -99,6 +99,7 @@ class BottomBar extends Component<any, any> {
     const currentControlsWidth = this._getControlsWidth(resizeObserverEntry.target as HTMLElement);
     this._maxControlsWidth = Math.max(this._maxControlsWidth, currentControlsWidth);
     if (barWidth !== this.currentBarWidth) {
+      window.dispatchEvent(new Event('bottomBarClientRectUpdated'));
```

Comment:
`@withEventManager` decorator adds eventManager into component that you can use instead of dispatching events directly to window. The issue here is next: if on page you have more than one Kaltura player events gonna be handled by every of them instead of specific player, where bottomBar size changed